.lando.yml

```diff
@@ -5,33 +5,55 @@ recipe: wordpress
 config:
   php: '7.2'
   via: nginx
-  webroot: trunk
+  webroot: public
   database: mariadb
   xdebug: false
 
+  conf:
+    server: config/wordpress.conf
+
```

Comment:
I believe this can be removed entirely since the `wordpress.conf` bundled here is no different from https://github.com/lando/lando/blob/master/plugins/lando-recipes/recipes/wordpress/wordpress.conf